### DISCUSSION TITLE
How do I remove top borders on my h2 section headers in a quarto book?

### DISCUSSION THREAD

**Discussion by jbrick55** [quarto-dev/quarto-cli#7764] [Q&A]
### Description

Basically, there is a horizontal (light gray) line above all h2 in my quarto book. There is also a border below the heading which is fine.
This seems like it should be easy with css but I am unable to remove it. I've tried css for container, level2, anchor, etc.. with border-bottom set to none... to no avail.  

Thanks for any advice.

**Comment by mcanouil** [comment]
I don't know what you've tried, but in this case your favourite browser is here to help.
(note that this is Bootstrap)

https://github.com/quarto-dev/quarto-cli/assets/8896044/c9b574ef-e7e4-447c-8fc6-da212208802c

````qmd
---
title: "Quarto Playground"
format: html
include-in-header:
  text: |
    <style type="text/css">
      h2, .h2 {
        border-bottom: none;
      }
    </style>
---

This is a playground for Quarto.

## Hello World
````

**Reply by jbrick55** [reply]
The video was very helpful... as I now know how to 'copy rule'!

However, I'm still unable to make this work. I'm trying to remove a 'border-top' of an h2, but the basic 'border-top: none' doesn't work. There are several classes when I inspect the header area but I am unable to make a style remove the border. 

I thought maybe it was putting a bottom border on the 'section' so it appear above the next h2. I tried..
`.section {
  border-bottom: none;
  border-top: none;
}`
 and that didn't work either.
Here's an image showing what I'm trying to remove.
<img width="705" alt="topborder" src="https://github.com/quarto-dev/quarto-cli/assets/54671911/3094487b-60ec-485b-a348-23b50a7fe8bd">

**Reply by mcanouil** [reply]
Unfortunately, if you don't share a reproducible example there is nothing that can be done to help.
Also, what I've shown works for all parts you only need to target/point it.

**Reply by jbrick55** [reply]
I'm sorry for wasting your time. From the inspect window I was able to see they were just horizontal rules that were added after each 'section' for some reason. I can't find the rule that controls this but I just set my hr to `display: none` and it worked.
I appreciate your video and what it taught me about copying the CSS rules.

**Reply by OlalObeja** [reply]
> I don't know what you've tried, but in this case your favourite browser is here to help. (note that this is Bootstrap)
> 
>  Screen.Recording.2023-12-01.at.22.09.08.mov 
> ```
> ---
> title: "Quarto Playground"
> format: html
> include-in-header:
>   text: |
>     <style type="text/css">
>       h2, .h2 {
>         border-bottom: none;
>       }
>     </style>
> ---
> 
> This is a playground for Quarto.
> 
> ## Hello World
> ```

Thanks @mcanouil some years later...I still found what you shared useful to fix the same issue. Thanks!
